wagtailinventory/templates/wagtailinventory/search.html

```diff
@@ -5,16 +5,36 @@
 
 {% block titletag %}{% trans "Block Inventory" %}{% endblock %}
 
+{% block extra_css %}
+<style>
+.wagtail-inventory-select .select {
+    display: inline-block;
+}
+
+.wagtail-inventory-select .field-content {
+    width: 100%;
+}
+
+.wagtail-inventory-select .select-has {
+    width: 175px;
+}
+
+.wagtail-inventory-select .select-block {
+    width: 50%;
+}
+</style>
+{% endblock %}
+
 {% block content %}
 {% trans "Block Inventory" as header_str %}
 {% include "wagtailadmin/shared/header.html" with title=header_str icon="placeholder" %}
 
 <form class="nice-padding" action="{% url 'wagtailinventory:search' %}">
 	{{ formset.management_form }}
   {% for form in formset %}
-    <div>
-      {{ form.has }}
-      {{ form.block }}
+    <div class="wagtail-inventory-select">
+      {% include "wagtailadmin/shared/field.html" with field=form.has show_label=False field_classes="select-has" %}
+      {% include "wagtailadmin/shared/field.html" with field=form.block show_label=False field_classes="select-block" %}
     </div>
   {% endfor %}
   <input type="submit" value="Find matching pages" style="width: 240px; padding: 10px; margin: 10px 0" />
```

Comment:
Making the button a little nicer while we're here.

```suggestion
  <input type="submit" value="Find matching pages" class="button" style="margin: 10px 0 20px;" />
```

![Image of button rendered with these suggested changes](https://user-images.githubusercontent.com/1044670/105226059-7a649480-5b2d-11eb-81d0-608839255325.png)